src/apps/bristleback_apps/aanderaa/user_code/user_code.cpp

```diff
@@ -162,36 +189,52 @@ void setup(void) {
   configASSERT(userConfigurationPartition);
   userConfigurationPartition->getConfig("currentAggPeriodMin", strlen("currentAggPeriodMin"),
                                         current_agg_period_min);
+  userConfigurationPartition->getConfig("nSkipReadings", strlen("nSkipReadings"),
+                                        n_skip_readings);
+  userConfigurationPartition->getConfig("readingIntervalMs", strlen("readingIntervalMs"),
+                                        reading_interval_ms);
+  userConfigurationPartition->getConfig("payloadWdToS", strlen("payloadWdToS"),
+                                        payload_wd_to_s);
+  userConfigurationPartition->getConfig("plUartBaudRate", strlen("plUartBaudRate"),
+                                        baud_rate);
+
+  max_readings_in_agg = (((uint64_t)CURRENT_AGG_PERIOD_MS / reading_interval_ms) + N_SAMPLES_PAD);
 
   aanderaaTopicStrLen = createAanderaaDataTopic();
   // Allocate memory for pressure data buffer.
   for (uint8_t i = 0; i < NUM_PARAMS_TO_AGG; i++) {
-    current_data[i].initBuffer(MAX_CURRENT_SAMPLES);
+    current_data[i].initBuffer(max_readings_in_agg);
   }
   stats_print_buffer = static_cast<char *>(
       pvPortMalloc(sizeof(char) * 1500)); // Trial and error, should be plenty of space
   // Initialize our LineParser, which will allocated any needed memory for parsing.
   parser.init();
   // Setup the UART – the on-board serial driver that talks to the RS232 transceiver.
   PLUART::init(USER_TASK_PRIORITY);
-  // Baud set per expected baud rate of the sensor.
-#ifdef AANDERAA_BLUE
-  PLUART::setBaud(115200);
-#else
-  PLUART::setBaud(9600);
-#endif
```

Comment:
Does 9600 baud work for both models or can we get rid of all the code referencing the blue?

Reply:
9600 baud is actually the default for both models - 115200 ended up being captured in the `#ifdef AANDERAA_BLUE` because the baud on the 1 Blue sensor we had had been adjusted to 115200 by someone before it made it to Russ.